<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1079</height>
      </rect>
    </property>
    <property name="font">
      <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
      </font>
    </property>
    <property name="windowTitle">
      <string>MainWindow</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <widget class="QSplitter" name="splitter2">
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QGraphicsView" name="roiGraphicsView">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize">
                <size>
                  <width>320</width>
                  <height>0</height>
                </size>
              </property>
              <property name="styleSheet">
                <string notr="true"/>
              </property>
            </widget>
            <widget class="QSplitter" name="splitter1">
              <property name="orientation">
                <enum>Qt::Vertical</enum>
              </property>
              <widget class="GraphicsView" name="roiDetailGraphicsView">
                <property name="maximumSize">
                  <size>
                    <width>16777215</width>
                    <height>16777215</height>
                  </size>
                </property>
                <property name="baseSize">
                  <size>
                    <width>0</width>
                    <height>800</height>
                  </size>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
              <widget class="QWidget" name="infoPanel">
                <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                    <widget class="QSplitter" name="splitter">
                      <property name="sizePolicy">
                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                          <horstretch>0</horstretch>
                          <verstretch>1</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                      </property>
                      <widget class="QTextEdit" name="annotationXML">
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                            <horstretch>1</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <!-- Set read-only -->
                        <property name="readOnly">
                          <bool>true</bool>
                        </property>
                        <property name="font">
                          <font>
                            <family>MS Shell Dlg 2</family>
                            <pointsize>12</pointsize>
                            <weight>50</weight>
                            <bold>false</bold>
                          </font>
                        </property>
                      </widget>
                      <widget class="QListWidget" name="imageInfoList">
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                            <horstretch>1</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="font">
                          <font>
                            <family>MS Shell Dlg 2</family>
                            <pointsize>12</pointsize>
                            <weight>50</weight>
                            <bold>false</bold>
                          </font>
                        </property>
                      </widget>
                    </widget>
                  </item>
                </layout>
              </widget>
            </widget>
          </widget>
        </item>
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
              <widget class="QPushButton" name="quitButton">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="text">
                  <string>QUIT</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="label_3">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="text">
                  <string>GUI Style</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QComboBox" name="styleComboBox">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <item>
                  <property name="text">
                    <string>DarkBreeze</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>DarkStyle</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>Default</string>
                  </property>
                </item>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="Zoom">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="text">
                  <string>Zoom</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QSpinBox" name="zoomSpinBox">
                <property name="maximumSize">
                  <size>
                    <width>60</width>
                    <height>16777215</height>
                  </size>
                </property>
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="minimum">
                  <number>20</number>
                </property>
                <property name="maximum">
                  <number>200</number>
                </property>
                <property name="singleStep">
                  <number>20</number>
                </property>
                <property name="value">
                  <number>60</number>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                  <widget class="QCheckBox" name="hideLabeled">
                    <property name="font">
                      <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text">
                      <string>Hide Verified</string>
                    </property>
                    <property name="checked">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QCheckBox" name="hideUnlabeled">
                    <property name="font">
                      <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text">
                      <string>Hide Unverified</string>
                    </property>
                    <property name="checked">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="sortButton">
                    <property name="font">
                      <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text">
                      <string>SORT</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="openVideo">
                    <property name="font">
                      <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="toolTip">
                      <string>Open the relevant video in a web browser, if available (Ctrl-V)</string>
                    </property>
                    <property name="text">
                      <string>OPEN VIDEO</string>
                    </property>
                    <property name="shortcut">
                      <string>Ctrl+V</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <spacer name="horizontalSpacer">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Maximum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>200</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
            <item>
              <widget class="QLabel" name="label">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="text">
                  <string>Class Label</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QComboBox" name="labelComboBox">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="toolTip">
                  <string>The concept label to apply to annotations</string>
                </property>
                <property name="editable">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QComboBox" name="partComboBox">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="toolTip">
                  <string>The part label to apply to annotations</string>
                </property>
                <property name="editable">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="labelSelectedButton">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="toolTip">
                  <string>Apply concept and part labels to selected localizations</string>
                </property>
                <property name="text">
                  <string>LABEL</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="clearSelections">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="toolTip">
                  <string>Clear all current selections</string>
                </property>
                <property name="text">
                  <string>CLEAR SELECTIONS</string>
                </property>
                <property name="shortcut">
                  <string>Ctrl+C</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="verifySelectedButton">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="toolTip">
                  <string>Verify selected localizations</string>
                </property>
                <property name="text">
                  <string>VERIFY</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="unverifySelectedButton">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="toolTip">
                  <string>Unverify selected localizations</string>
                </property>
                <property name="text">
                  <string>UNVERIFY</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="discardButton">
                <property name="font">
                  <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>
                <property name="toolTip">
                  <string>Delete selected localizations</string>
                </property>
                <property name="text">
                  <string>DELETE</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
    <widget class="QStatusBar" name="statusBar"/>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>1920</width>
          <height>19</height>
        </rect>
      </property>
    </widget>
  </widget>
  <customwidgets>
    <customwidget>
      <class>GraphicsView</class>
      <extends>QGraphicsView</extends>
      <header>pyqtgraph</header>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections>
    <connection>
      <sender>quitButton</sender>
      <signal>pressed()</signal>
      <receiver>MainWindow</receiver>
      <slot>close()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>563</x>
          <y>861</y>
        </hint>
        <hint type="destinationlabel">
          <x>563</x>
          <y>450</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>
